src/main/riddl/Order/Order.riddl

```diff
@@ -141,13 +140,13 @@ context OrderContext is {
             and "yield OrderStatusUpdated event"
           }
           example InProcessOrderIllegalStateTransition is {
-            given "order is transitioning from InProcess to Draft, Pending, Cancelled, or Delivered"
+            given "order is transitioning from InProcess to InProcess, Draft, Pending, Cancelled, or Delivered"
             then error "Draft orders may not transition to desired state."
           }
         }
-        on command UpdateOrderInfo {then error "InProcess Orders may not be modified"}
+        on command UpdateOrderInfo {then error "InProcess Orders may not be modified."}
         on command CancelOrder {then error "InProcess Orders may not be cancelled."}
-        on query GetOrderInfo {then "yeild OrderInfoResult message"}
+        on query GetOrderInfo {then "yield OrderInfoResult message"}
```

Comment:
Either I copy/pasted a lot or I just can't spell yield... Either way, thanks for cleaning up!